src/com/amazon/ionschema/Import.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.ionschema
+
+/**
+ * An Import represents all the types imported by a Schema
+ * from one schema id.
+ *
+ * Note that multiple ISL imports referencing the same schema id
+ * (each importing/aliasing an individual type) are represented by a
+ * single Import instance.
+ */
+interface Import {
+    /**
+     * The id of the referenced schema.
+     */
+    val id: String
+
+    /**
+     * Returns the schema referenced by the import.
+     */
+    fun getSchema(): Schema
+
+    /**
+     * Returns the requested type, if present in this import;
+     * otherwise returns null.  If a type is aliased (via "as")
+     * by the import, the type will only be returned from this
+     * method by its alias (not the imported type's original name).
+     */
+    fun getType(name: String): Type?
+
+    /**
+     * Returns an iterator over the types imported by this Import.
+     * Callers must not rely on any particular ordering, as Types
```

Comment:
Would it be better to return a Set instead. Also the Types returned they hold the alias names from the import (`as: x`). 

If `Type` allows me to extract both names (original and aliased) then please ignore my question. 

I am mostly asking to make sure we document what these names are so as to not surprise client code.

Reply:
I'm following the example set by [Schema.getTypes()](https://github.com/amzn/ion-schema-kotlin/blob/master/src/com/amazon/ionschema/Schema.kt#L51), as well as the new `Schema.getImports(): Iterator<Import>` method added in this PR.

For Set-like behavior, I'd expect an `Import` object's `getType(name)` method to be sufficient (the Import object effectively *is* the Set).  Whereas the style of `getTypes()` assumes you want to iterate over the types.

As for aliasing, the approach is to only expose an aliased type by its alias, not its original name.  So:

```
val type = import.getType(name)
assertEquals(name, type.name)
```

There is a catch, however.  The `IonValue` provided by `type.isl` *does not* take aliasing into account--it is the original ISL of the type.